Formula/libbladerf.rb

```diff
@@ -1,10 +1,14 @@
 class Libbladerf < Formula
   desc "USB 3.0 Superspeed Software Defined Radio Source"
   homepage "https://nuand.com/"
-  url "https://github.com/Nuand/bladeRF/archive/2018.08.tar.gz"
-  sha256 "6288c230dad26e32236a4b60f0b14c129e6fa0ad91bcf1c40abe8789b352e51f"
   head "https://github.com/Nuand/bladeRF.git"
 
+  stable do
+    url "https://github.com/Nuand/bladeRF.git",
+        :tag      => "2019.07",
+        :revision => "991bba2f9c4d000f000077cc465878d303417e26"
+  end
+
```

Comment:
Please don't move this into a stable block but keep it in the same place it was before.